six/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "six",
+    "assets": [
+      {
+        "description": "The native staking token of the SIX Protocol.",
+        "denom_units": [
+          {
+            "denom": "usix",
+            "exponent": 0
+          },
+          {
+            "denom": "six",
+            "exponent": 6
+          }
+        ],
+        "base": "usix",
+        "name": "Six token",
+        "display": "six",
+        "symbol": "six",
```

Comment:
Please capitalize to SIX

Reply:
Which one all of them?
1. chain_name
2. name
3. display
4. symbol